src/services/dashboards/dashboard-create/DashboardCreatePage.vue

```diff
@@ -87,25 +87,27 @@ export default {
 
         const state = reactive({
             dashboardScope: DASHBOARD_SCOPE.DOMAIN as DashboardScope,
-            dashboardViewerType: DASHBOARD_VIEWER.PRIVATE as DashboardViewer,
+            dashboardViewerType: DASHBOARD_VIEWER.PUBLIC as DashboardViewer,
         });
 
         const handleClickCreate = () => {
             let _dashboardTemplate;
             if (state.dashboardScope === DASHBOARD_SCOPE.PROJECT) {
                 _dashboardTemplate = {
                     ...dashboardTemplate.value,
-                    viewers: state.dashboardViewerType,
+                    dashboard_id: undefined,
                     project_id: dashboardProject.value?.id ?? '',
+                    viewers: state.dashboardViewerType,
                 };
             } else {
                 _dashboardTemplate = {
                     ...dashboardTemplate.value,
+                    dashboard_id: undefined,
                     viewers: state.dashboardViewerType,
                 };
             }
 
-            dashboardDetailStore.setDashboardInfo(_dashboardTemplate);
+            dashboardDetailStore.setDashboardInfo(_dashboardTemplate, undefined);
```

Comment:
suggestion: How about this?

```suggestion
            dashboardDetailStore.setDashboardInfo(_dashboardTemplate);
            dashboardDetailStore.dashboardId = undefined;
```

Reply:
Oh it looks good~!